include/deal.II/grid/grid_refinement.h

```diff
@@ -300,6 +301,65 @@ namespace GridRefinement
                                const VectorType            &criteria,
                                const unsigned int          order=2);
 
+
+  /**
+   * Return the highest ratio among the dimensions of a @p cell. Moreorer,
+   * return the dimension relative to the highest elongation.
+   *
+   * @param[in] cell : an iterator pointing to the cell.
+   *
+   * @return  a std::pair<unsigned int, double> such that the @p first value
+   * is the dimension of the highest elongation and the @p second value is the
+   * ratio among the dimensions of the @p cell
+   *
+   * @note This function works in dimension 2 and 3.
+   */
+  template<int dim, int spacedim>
+  std::pair<unsigned int, double>
+  get_longest_direction(typename Triangulation<dim, spacedim>::active_cell_iterator cell);
+
+  /**
+   * Remove hanging nodes from a grid. @p isotropic (default) allows to reproduce
+   * the proportion of each cell in child cells.
+   * Since this procedure is possibly infinite a max number of iteration is
+   * provided
+   *
+   * @param[in,out] tria : tria to refine
```

Comment:
Spell out "tria" to "The triangulation..."

Same below: Start a complete sentence after the name of the parameter.